app/views/entries/show.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <div class="section">
   <p>
-    Belongs to User <%= @list.user_id %> within List <%= @entry.list_id%>
+    Belongs to User <%= @entry.list.user_id %> within List <%= @entry.list_id %>
```

Comment:
Maybe it would be better to do `@entry.list.user.username` and `@entry.list.title` respectively?

Reply:
You  could also make the List name a link back to the actual List.